spec/libraries/duplicator_spec.rb

```diff
@@ -612,14 +613,17 @@ def create_ar_graph
 
             it 'duplicates cyclic graph' do
               duplicator = Duplicator.new
-              dup_c1 = duplicator.duplicate(@car1)
-              dup_c1.save
 
-              dup_c3 = dup_c1.children[0].children[0]
-              dup_c5 = dup_c3.children[0]
-              dup_c4 = dup_c1.children[0].children[1]
+              # Check that number of objects increased
+              expect do
+                @dup_c1 = duplicator.duplicate(@car1)
+                @dup_c1.save
+              end.to change{ComplexActiveRecord.count}.by(6)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.